contribs/drt-extensions/src/test/java/org/matsim/contrib/drt/extension/edrt/run/RunEDrtScenarioIT.java

```diff
@@ -129,7 +129,7 @@ void testMultiModeDrtDeterminism() {
 
 		controller.run();
 
-		assertEquals(1926, tracker.passengerPickupEvents);
+		assertEquals(1919, tracker.passengerPickupEvents);
```

Comment:
This small difference is explained by the fact that the previously existing hub return timeout was not inclusive, i.e. a timeout of "60" means that at least 61 seconds had to pass. This is now changed such that it is "at least" 60s. Otherwise, if the timeout is set to "0" you would still need to have a stay task of at least 1s, which means one can't really model a non-limiting timeout